.github/workflows/rust.yml

```diff
@@ -505,15 +494,6 @@ jobs:
     - name: Format (manifest)
       run: cargo verify-project
 
-    - name: Run cargo audit
```

Comment:
It looks like this PR removes auditing for both examples + reactors (though keeps the SDK itself `cargo audit`ed). Is that intended? If so, could you leave a comment as to express this happened and indicate that it is unnecessary?

Reply:
It's the same reasoning as to why they were excluded in local tests in #280 (i.e. the cost of building those tools doesn't provide too much value in examples, so it's not worth building them if they're not pre-installed as in full `ubuntu-22.04` images on GitHub Actions).

With that in mind, I moved them to the `audit` and `outdated` jobs, since we build those tools for the SDK already.